legacy/application/models/Preference.php

```diff
@@ -1399,6 +1399,16 @@ public static function setRadioPageDisplayLoginButton($value)
         self::setValue('radio_page_display_login_button', $value);
     }
 
+    public static function getScheduleTrimOverbooked()
+    {
+        return boolval(self::getValue('schedule_trim_overbooked'), false);
```

Comment:
For your information, the false should be an argument of the getValue function, it provides the default value.

Already fixed it.